### ISSUE TITLE
I'd like to be able to run interactive commands.

### ISSUE THREAD

**Issue by AlbertClo** [soloterm/solo#15]
Sometimes a command might need some interaction from time to time. E.g. when I'm running Storybook, and the default port for it is occupied, it asks if you want to run it on another port.

Currently I don't see a way to tell Storybook to continue running.

![Screenshot_20241110_201236](https://github.com/user-attachments/assets/3dabbe5b-e188-4919-a971-2274958ebff6)

**Comment by simonhamp** [comment]
You have a couple of options here with Storybook:

1. You could be explicit about the port in your command. I haven't tried this but something like may work e.g. `npm run storybook -- -p6007`
2. You can try passing the `-y` flag which should answer `yes` to any prompts: `npm run storybook -- -y`

Note the `--` before passing Storybook-specific flags/parameters - this is how the shell & npm know to pass these down to storybook itself. Otherwise you'd have to write out the full command from your `package.json->scripts->storybook`

**Comment by AlbertClo** [comment]
Yeah, I could do that. But this just an example of a general problem. There are many other reasons to want to be able to drop into interactive mode.

**Comment by aarondfrancis** [comment]
Working on this in #4!

**Comment by antoniofrignani** [comment]
Another use case for this is to have a Tinker tab opened to test simple commands directly from within it.

Actually, if i try to add a Tinker tab, it dies poorly 😅

**Comment by aarondfrancis** [comment]
Done! You'll need to update to 0.3.0, which requires removing `aaronfrancis/solo` and installing `soloterm/solo`, but then you can do it!

**Comment by AlbertClo** [comment]
Excellent. I'll update and try it soon.

**Comment by jhonoryza** [comment]
to add tinker add this
```php
    'commands' => [

        'Tinker' => Command::make('Tinker', 'php artisan tinker', true)->interactive(),
   ]
```
